mod/state-transition/pkg/core/state_processor.go

```diff
@@ -197,34 +198,26 @@ func (sp *StateProcessor[
 ]) ProcessSlots(
 	st BeaconStateT, slot math.Slot,
 ) (transition.ValidatorUpdates, error) {
-	var (
-		validatorUpdates      transition.ValidatorUpdates
-		epochValidatorUpdates transition.ValidatorUpdates
```

Comment:
nit: reduced `epochValidatorUpdates` scope (it's only used when processing epochs)